lib/resolvers/project-version.js

```diff
@@ -72,6 +72,16 @@ module.exports = ({ models }) => async ({ action, data, id }, transaction) => {
 
     const { protocolId, retrospectiveAssessment } = data;
     const { conditions, reminders } = extractReminders(version, data.conditions);
+    const reminderIds = reminders.map(reminder => reminder.id);
+
+    // Compares the deadlines in the state to the ones in the database, if in database but no longer in state flag as deleted
+    const remindersFromDb = await Reminder.query().where({model_id: version.projectId});
```

Comment:
```suggestion
    const remindersFromDb = await Reminder.query().where({modelId: version.projectId});
```
Even though the DB column names are snake_case the actual schema model props are camelCase, so if we're using objection then it's best to use camelCase. 

it's only if you're building a raw knex query then you'll probably need to use snake_case.